src/components/Checkbox/index.test.js

```diff
@@ -1,48 +1,52 @@
 import React from 'react';
-import { shallow } from 'enzyme';
+import { render, fireEvent, screen } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect'
 
 import Checkbox from '.';
 
-// uuid will break snapshots, so we must mock it.
-//
-// look at me, i'm universally unique, I'm _soooooo special_
-jest.mock('uuid/v4', () => jest.fn().mockReturnValue('mock-universal-unique-identifier'));
-
 describe('Checkbox component', () => {
-
-  it('matches snapshot (without label)', () => {
-    const wrapper = shallow(<Checkbox name="snapshot_checkbox" showLabel={false} label="snapshot checkbox" />);
-    expect(wrapper).toMatchSnapshot();
-  });
-
-  it('matches snapshot (with label)', () => {
-    const wrapper = shallow(<Checkbox label="Snapshot Label" name="snapshot_checkbox" />);
-    expect(wrapper).toMatchSnapshot();
-  });
-
-  it('matches snapshot (disabled)', () => {
-    const wrapper = shallow(<Checkbox disabled label="Snapshot Label" name="snapshot_checkbox" />);
-    expect(wrapper).toMatchSnapshot();
+  it('passes the `disabled` prop', () => {
+    const { container } = render(<Checkbox label="change callback" disabled name="disabled-test" />);
+    const input = screen.getByLabelText('change callback');
+    expect(input.disabled).toBeTruthy();
+
+    fireEvent.click(input);
+    expect(input.checked).toBeTruthy();
+    expect(container.querySelector('.fdsCheckable')).toHaveClass('fdsCheckable--disabled')
```

Comment:
```suggestion
    expect(screen.getByRole('checkbox')).toHaveAttribute('disabled'
```

Reply:
What do you think of this suggestion?

Reply:
I was following the styling already existent in other tests for the class check. I think your suggestion is a slightly different expectation that has value as well.